crypto-serde/Cargo.toml

```diff
@@ -31,3 +31,8 @@ toml = "0.5"
 [features]
 default = ["alloc"]
 alloc = ["base16ct/alloc", "serde/alloc"]
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
+targets = []
```

Comment:
I noticed that is something that RustCrypto isn't doing in other projects (I think). Should I remove it?
See:
- https://blog.rust-lang.org/2020/03/15/docs-rs-opt-into-fewer-targets.html
- https://docs.rs/about/metadata

Reply:
Looks good. We should probably do it everywhere we don't have target-dependent features.